mozdownload/scraper.py

```diff
@@ -51,6 +51,9 @@
                            'win32': 'exe',
                            'win64': 'exe'}
 
+DEFAULT_BRANCHES = {'thunderbird': 'comm-central'
```

Comment:
I would suggest you also add firefox and fennec. Just to make it better configurable at some point.